analyticsclient/client.py

```diff
@@ -41,6 +42,7 @@ def __init__(self, base_url, auth_token=None, timeout=0.25):
 
         self.status = Status(self)
         self.courses = lambda course_id: Course(self, course_id)
+        self.users = lambda user_id: User(self, user_id)
```

Comment:
@bradenmacdonald Since the data API URLs now require [passing a username](https://github.com/open-craft/edx-analytics-data-api/blob/user-profiles/analytics_data_api/v0/urls/users.py#L6) instead of an ID I think it would be good to change the terminology in the client as well.